src/playground/components/Configuration.js

```diff
@@ -133,6 +129,8 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
         );
     };
 
+    const configFileContent = `${options.languageOptions?.sourceType === "module" ? "export default" : "module.exports ="} ${JSON.stringify([options], null, 4)};`;
```

Comment:
I don't think this is a safe assumption to make, at least, not based on the feedback we received about wanting `eslint.config.cjs` and `eslint.config.mjs` support. It's probably better to offer the choice of download format.

Reply:
Updated, now users can choose the format.
<img width="380" alt="Screenshot 2024-01-25 at 3 48 39 PM" src="https://github.com/eslint/eslint.org/assets/46647141/4da2734f-c79e-4586-982b-5143bc6f939a">

Reply:
This option is too far away from the download button. Maybe we can make it closer to the download button and remove the text about formatting on the button. Turning it into two buttons would be nice.

Reply:
I've moved the selector closer to the button.

<img width="378" alt="Screenshot 2024-01-26 at 4 38 58 PM" src="https://github.com/eslint/eslint.org/assets/46647141/c4251d02-78d7-463d-88ae-f3943ba4fcac">